galleries/examples/axes_grid1/demo_colorbar_with_axes_divider.py

```diff
@@ -1,7 +1,7 @@
 """
-============================
-Colorbar with `.AxesDivider`
-============================
+==================================
+Colorbar with :obj:`~.AxesDivider`
+==================================
```

Comment:
I think we should avoid the link in the title altogether.
```suggestion
=============================
Colorbar with ``AxesDivider``
=============================
```

Reply:
I agree, the headings turn into their own links and links in links gets funny.